src/OVAL/probes/probe/worker.c

```diff
@@ -969,6 +969,8 @@ SEXP_t *probe_worker(probe_t *probe, SEAP_msg_t *msg_in, int *ret)
 	probe_offline_mode_function_t offline_mode_function = probe_table_get_offline_mode_function(probe->subtype);
 	if (offline_mode_function != NULL) {
 		probe->supported_offline_mode = offline_mode_function();
+	} else {
+		probe->supported_offline_mode = PROBE_OFFLINE_NONE;
```

Comment:
Should add a debug message here if possible as well.

Reply:
In the worker thread function there is a warning already:
```c
        if (probe->supported_offline_mode == PROBE_OFFLINE_NONE) {
            dW("Requested offline mode is not supported by %s probe.", oval_subtype_get_text(probe->subtype));
            *ret = 0;
            return probe_cobj_new(SYSCHAR_FLAG_NOT_APPLICABLE, NULL, NULL, NULL);
```

Reply:
Should it be a warning, thought? Maybe info?